src/ui/Theme/component.tsx

```diff
@@ -23,7 +23,14 @@ const GlobalStyle = createGlobalStyle`
 
   body {
     font-size: 1.6rem;
-    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif
+    font-family: 'PT Mono', monospace;
```

Comment:
We should keep the default fonts after `'PT Mono'`.

Reply:
Sure. But this will change as soon as we start working on the design :)